infrastructure-scripts/claude_start.sh

```
@@ -0,0 +1,1640 @@
+
+    # Alternative: Check for any active task agents
+    if tmux list-sessions 2>/dev/null | grep -q "task-agent-"; then
+        return 0
+    fi
+
+    return 1
+}
+
+# Function to start orchestration in background
+start_orchestration_background() {
+    local SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)